packages/app/src/server/util/slack.js

```diff
@@ -131,7 +131,7 @@ export const prepareSlackMessageForComment = (comment, user, appTitle, siteUrl,
   const text = `:speech_balloon: ${user.username} commented on ${pageUrl}`;
 
   const message = {
-    channel: (channel != null) ? `#${channel}` : undefined,
+    channel: (channel != null) ? `${channel}` : undefined,
```

Comment:
Sorry, one more point.

UserNotificationService uses this method.
Please confirm this change does not cause any problem.

Reply:
I've tested it first. but to make sure I'm submitting the latest test results as follows:
### Coment
![image](https://user-images.githubusercontent.com/92426728/171607183-8730c5d1-53e6-47e0-90c8-147b7d5f01de.png)

### Notification
- I entered two Slack channels and both received the same notifications 
![image](https://user-images.githubusercontent.com/92426728/171607288-cda3cef8-7184-4901-adbe-865372c170d2.png)